src/disco/components/Addon.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+
+import InstallButton from './InstallButton';
+
+import 'disco/css/Addon.scss';
+
+
+export default class Addon extends React.Component {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    type: PropTypes.string.isRequired,
+    heading: PropTypes.string.isRequired,
+    subHeading: PropTypes.string,
+    editorialDescription: PropTypes.string.isRequired,
+  }
+
+  getLogo() {
+    const { id } = this.props;
+    const imageURL = `https://addons-dev-cdn.allizom.org/user-media/addon_icons/0/${id}-64.png?modified=1388632826`;
```

Comment:
These URL formats should probably go into the config. It would probably be nice to have them returned by the API instead though.